### PR TITLE
Scenarios select mining pool

### PR DESCRIPTION
**Pull request by Oscar-Pepper**

### CODE REVIEW

**Comment by Oscar-Pepper** [comment]
superceeds #592

**Comment by fluidvanadium** [comment]
fox crap, this is great stuff and its gonna break the test im working on

**Review by fluidvanadium — DISMISSED** [review]

**Review by fluidvanadium — DISMISSED** [review]

**Review by fluidvanadium — DISMISSED** [review]

**Review by fluidvanadium — APPROVED** [review]
verbose... but its progreses

File: zingolib/src/wallet.rs [hunk 1]
```diff
@@ -86,16 +86,16 @@ pub struct SendProgress {
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub enum Pool {
-    Sapling,
     Orchard,
+    Sapling,
```
Comment (fluidvanadium) [inline zingolib/src/wallet.rs:90]: i typically write such an enum in the opposite order: transparent .. sapling .. orchard...

**Review by fluidvanadium — APPROVED** [review]

**Pull request merged**